test/fixtures/umbrella/mix.exs

```diff
@@ -0,0 +1,12 @@
+defmodule Umbrella.Mixfile do
```

Comment:
I think it is easier to just create this in the test using `File.write!` instead of duplicating the whole in_fixture machine. Wdyt?